pyccel/naming/cudanameclashchecker.py

```diff
@@ -16,6 +16,10 @@ class CudaNameClashChecker(LanguageNameClashChecker):
     verify that they do not cause name clashes. Name clashes may be due to
     new variables, or due to the use of reserved keywords.
     """
+
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
:| these warnings are a bit annoying on cases like this. Can you do the same as here:
https://github.com/pyccel/pyccel-cuda/blob/bd7351493e3ae2c0947b1d2fb92605360db4de08/pyccel/ast/datatypes.py#L66-L69

but in the superclass please